ddl/column.go

```diff
@@ -727,6 +740,26 @@ func (w *worker) doModifyColumn(t *meta.Meta, job *model.Job, newCol *model.Colu
 	return ver, nil
 }
 
+func checkAndApplyNewAutoRandomBits(t *meta.Meta, schemaID int64, tblInfo *model.TableInfo,
+	newCol *model.ColumnInfo, oldName *model.CIStr, newAutoRandBits uint64) error {
+	newLayout := autoid.NewAutoRandomIDLayout(&newCol.FieldType, newAutoRandBits)
+	currentIncBitsVal, err := t.GetAutoRandomID(schemaID, tblInfo.ID)
```

Comment:
It seems no lock here. Is it possible that `currentIncBitsVal` increases by 1 just after you read it?

Reply:
@djshow832 Updated. I added `GenAutoRandomID` before `GetAutoRandomID`. Since these are running in a transaction, if it increases by 1, the write conflict error is thrown.